.github/workflows/native-macos.yml

```diff
@@ -10,34 +10,36 @@ on:
 
 jobs:
   build-macos:
-    if: false && (github.event_name == 'push' || github.event.pull_request.merged == true)
-    name: macOS ${{ matrix.osversion }} ${{ matrix.abi == '64' && ' ' || matrix.abi }} ${{ matrix.cc == 'gcc' && ' ' || matrix.cc }} ${{ matrix.nroff == 'no-roff' && matrix.nroff || ' ' }} ${{ matrix.cflags == '' && ' ' || 'no-obsolete' }} ${{ matrix.utmp == 'autoselect' && ' ' || matrix.utmp }}
+    if: true && (github.event_name == 'push' || github.event.pull_request.merged == true)
+    name: >
+      macOS
+      ${{ matrix.osversion }}
+      ${{ matrix.abi == '64' && ' ' || matrix.abi }}
+      ${{ matrix.cc == 'gcc' && ' ' || matrix.cc }}
+      ${{ matrix.nroff == 'no-roff' && matrix.nroff || ' ' }}
+      ${{ matrix.cflags == '' && ' ' || 'no-obsolete' }}
+      ${{ matrix.utmp == 'autoselect' && ' ' || matrix.utmp }}
     runs-on: macos-${{ matrix.osversion }}
     strategy:
       fail-fast: false
       matrix:
         osversion: ["12", "11"]
         abi: [64]
         cflags: ["-DDEPRECATED_FUNCTIONS_REMOVED"]
+        warnings: "-Wall -Wshadow -Werror=implicit-function-declaration -Werror=incompatible-library-redeclaration -Werror=parentheses -Werror=dangling-else -Werror=pointer-sign"
         cc: ["clang"]
         nroff: ["nroff", "no-roff"]
         utmp: ["autoselect", "force-old-utmp"]
     steps:
     - name: Checkout
       uses: actions/checkout@v4
-    - name: Test
-      id: test
-      env:
-        CC: ${{ matrix.cc }}
-        CFLAGS: "${{ matrix.cflags }} -O2 -fPIC -Wall -Wshadow -Werror=implicit-function-declaration ${{ matrix.cc == 'clang' && '-Werror=incompatible-library-redeclaration' || ' ' }} -Werror=parentheses -Werror=dangling-else -Werror=pointer-sign -Werror=pointer-sign"
-        NROFF: ${{ matrix.nroff }}
-        UTMP: ${{ matrix.utmp }}
```

Comment:
Always nice to have fewer boilerplate. The lines end-up slightly longer, but still readable.

Reply:
Yes, my feeling is that the the tradeoff is desirable: the job gets more glanceable than the line lengths get unreadable ;-)